src/apiRequests.js

```diff
@@ -1,6 +1,35 @@
 export const getData = endpoint => {
   return fetch(`http://localhost:3001/api/v1/${endpoint}`)
-    .then(response => {
-      return response.json()
-    });
+    .then(response => response.json())
+    .catch(error => console.log('getData fetch error', error));
+}
+
+
+export const postTrip = (newID, userID, destinationID, travelers, date, duration) => {
+  var myHeaders = new Headers();
+  myHeaders.append("Content-Type", "application/json");
+
+  var raw = JSON.stringify({
+    "id": parseInt(`${newID}`),
+    "userID": parseInt(`${userID}`),
+    "destinationID": parseInt(`${destinationID}`),
+    "travelers": parseInt(`${travelers}`),
+    "date": `${date}`,
+    "duration": parseInt(`${duration}`),
+    "status": "pending",
+    "suggestedActivities": []
+  });
+  console.log(raw);
+
+  var requestOptions = {
+    method: 'POST',
+    headers: myHeaders,
+    body: raw,
+    redirect: 'follow'
+  };
+
+  return fetch("http://localhost:3001/api/v1/trips", requestOptions)
+    .then(response => response.text())
+    .then(result => console.log(result))
+    .catch(error => console.log('error', error));
```

Comment:
Just don't know that I've seen it anywhere yet - wondering what the user may see to indicate if a request has not gone through if it fails when they try to make one?